front/lib/models/plan.ts

```diff
@@ -148,13 +149,6 @@ Subscription.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
-    status: {
```

Comment:
Let's reinstate that. Figuring out current subscription by date logic will make our life miserable I'm pretty sure. Let's consider dates as logging information and status as ground truth.

Reply:
Done!